students/BrianE/session04/MailRoom.py

```diff
@@ -26,13 +26,14 @@ def send_thankyou(donors):
             else:
                 donors[name] = [amount]
             email_message = f"Dear {name.strip()},\n" \
-                            f"    Thank you for your recent donation of ${amount}!\n\n" \
-                            f"    Sincerely,\n\n" \
-                            f"    FakeCorp, Inc"
+                f"    Thank you for your recent donation of ${amount}!\n\n" \
```

Comment:
if you want to put strings on multiple lines, you don't need an f before each one. And you ceratinly don't need an f for a string with no formatting in it. And you son't need the \ is it's in a parentheses. So this can be:

```
email_message = (f"Dear {name.strip()},\n" 
                  "    Thank you for your recent donation of ${amount}!\n\n" 
                  "    Sincerely,\n\n" 
                  "    FakeCorp, Inc")